src/apps/integrations/router.py

```diff
@@ -1,16 +1,16 @@
 from fastapi.routing import APIRouter
 from starlette import status
 
-from apps.integrations.api import create_integration, disable_integration, enable_integration
-from apps.integrations.loris.domain.loris_projects import LorisProjects
+from apps.integrations.api import create_integration, disable_integration, enable_integration, retrieve_integration
+from apps.integrations.domain import Integration
 from apps.shared.domain import Response
 from apps.shared.domain.response import AUTHENTICATION_ERROR_RESPONSES, DEFAULT_OPENAPI_RESPONSE
 
 router = APIRouter(prefix="/integrations", tags=["Integration"])
 
 
 router.post(
-    "/",
+    "/enable_integration",
```

Comment:
Why are you adding `/enable_integration` and `/disable_integration` again? Are you going to update these endpoints in a different task/pr?